4/server.js

```diff
@@ -0,0 +1,16 @@
+const { ApolloServer } = require('apollo-server');
+const { makeAugmentedSchema } = require('neo4j-graphql-js');
+const { typeDefs, resolvers } = require('./typedefs');
+const neo4j = require('neo4j-driver');
+
+const driver = neo4j.driver(
+    'bolt://localhost:7687',
+    neo4j.auth.basic('testuser', '1234')
```

Comment:
Please parameterize this. I have to change the source code when I want to try out your code.